ac-cider.el

```diff
@@ -87,7 +87,6 @@ Caches fetched documentation for the current completion call."
                   "\r" ""
                   (nrepl-dict-get (cider-nrepl-send-sync-request
                                    (list "op" "complete-doc"
-                                         "session" (nrepl-current-session)
```

Comment:
Even if you don't specify the session, the request is going to be evaluated in the context of a session on the server - it's definitely better for this to be the current session.